extractor/src/main/java/org/schabi/newpipe/extractor/playlist/PlaylistExtractor.java

```diff
@@ -21,6 +22,8 @@ public PlaylistExtractor(final StreamingService service, final ListLinkHandler l
 
     public abstract long getStreamCount() throws ParsingException;
 
+    public abstract Description getDescription() throws ParsingException;
```

Comment:
I think you should provide a default implementation returning `Description.EMPTY_DESCRIPTION`, like done for other methods below, as most of the implementations return this value.

Reply:
Would you still want this to be done? TobiGr added support for PeerTube and Soundcloud